### PR TITLE
Restructure maintenance chapter

### PR DESCRIPTION
**Pull request by tahliar**
### PR creator: Description

I restructured the Maintenance chapter to hopefully be a bit more straightforward. I mostly just moved content to new locations, but I did a little editing as well.

Original chapter: https://documentation.suse.com/sle-ha/15-SP4/html/SLE-HA-all/cha-ha-maintenance.html

New chapter: [cha-ha-maintenance_color_en.pdf](https://github.com/SUSE/doc-sleha/files/10256945/cha-ha-maintenance_color_en.pdf)

### PR creator: Are there any relevant issues/feature requests?

* jsc#DOCTEAM-831

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE-HA 15
  - [x] 15 next *(current `main`, no backport necessary)*
  - [x] 15 SP4
  - [x] 15 SP3
  - [x] 15 SP2
  - [x] 15 SP1
  - [x] 15
- SLE-HA 12
  - [ ] 12 SP5
  - [ ] 12 SP4

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

File: xml/ha_maintenance.xml [hunk 1]
```diff
@@ -96,97 +83,52 @@
...
+  <warning>
+   <title>Risk of data loss</title>
    <para>
-    If your aim is to move the services off the node in an orderly fashion
-    before shutting down or rebooting the node, proceed as follows:
+    If you need to do testing or maintenance work, follow the general steps
+    below.
    </para>
+   <para>
+    Otherwise you risk unwanted side effects, like resources not starting in an
```
Comment (dariavladykina) [inline xml/ha_maintenance.xml:98]:
```suggestion
    Otherwise, you risk unwanted side effects, like resources not starting in an
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please see some suggestions here. Thanks!

**Comment by dariavladykina** [comment]
Hi @tahliar, I've also spotted one typo in the text that wasn't meant for review, in this paragraph: 
When putting a cluster into maintenance mode, only the resource management by Pacemaker is disabled. Corosync and SBD are still functional. Depending on your *maintainence* tasks, this might lead to fence operations.

**Comment by tahliar** [comment]
Great, thank you @dariavladykina!

**Pull request merged**